docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -7096,6 +7096,152 @@ PI()
 PI()
 ```
 
+<<<
+[[pivot_function]]
+== PIVOT Function
+
+The PIVOT function transposes rows of a table into a column, aggregating data in the process of the rotation.
+
+PIVOT is a Trafodion SQL extension.
+
+`PIVOT (_source-column_[, DELIMITER '_delimiter_'])`
+
+[[syntax_description_of_pivot]]
+=== Syntax Description of PIVOT
+
+* `_source-column_`
+
++
+is a column containing rows to be rotated.
+
+* `_delimiter_`
+
++
+is a delimiter used to separate values in a column.
+
++
+NOTE: If the `DELIMITER '_delimiter_'` is omitted, the default delimiter is a comma.
+
++
+*Example*
+
++
+The table _vendor_ is as follows:
+
++
+```
+SELECT * FROM vendor;
+
+VENDOR_ID VENDOR_NAME    VENDOR_EMAIL           VENDOR_EMAIL_ID
+--------- -------------- ---------------------- ---------------
+      111 Tom            tom@hotmail.com        1
+      232 Jerry          jerry@hotmail.com      2
+      232 Jerry          jerry@gmail.com        3
+      367 Aven           aven@hotmail.com       4
+      367 Aven           aven@gmail.com         5
+      367 Aven           aven@outlook.com       6
+
+--- 6 row(s) selected.
+```
+
++
+```
+SELECT PIVOT(vendor_id), PIVOT(vendor_email) FROM vendor GROUP BY vendor_id;
```

Comment:
It might be more typical for a user here to do "select vendor_id, pivot(vendor_email) from vendor group by vendor_id".

Reply:
Thanks for your suggestion, I've corrected.